templates/web/pages/remove_products/remove_products.tt.html

```diff
@@ -0,0 +1,20 @@
+<!-- start templates/[% template.name %] -->
+
+[% IF action == 'display' %]
+
+    <p>[% lang("remove_products_from_producers_platform_description") %]</p>
+    <p>[% lang("this_action_cannot_be_undone") %]</p>
+
+    <!--Start the form-->
+    <form id="remove_products_form" method="post" onsubmit="return confirm('[% confirm_alert %]')" enctype="multipart/form-data">
```

Comment:
there is a possibility that in one of the languages contains a single quote, which would brea the confirm() javascript.

I think that you can use the squote filter from Template Toolkit to escape any single quotes: http://template-toolkit.org/docs/manual/VMethods.html#section_squote